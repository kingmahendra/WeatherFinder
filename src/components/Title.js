import React from 'react'

export const Title = () => (
    <div> 
        <h1 className="title-container__title" > Weather Finder </h1>
        <h3 className="title-container__subtitle"> Find out temperature, humidity and ohter weather conditions. </h3> 
   </div>  
)