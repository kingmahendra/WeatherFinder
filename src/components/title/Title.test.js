import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'

import { Title } from './Title'

describe('Title Component', ()=> {
    it('should render title', () => {
        const component = shallow(<Title />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})