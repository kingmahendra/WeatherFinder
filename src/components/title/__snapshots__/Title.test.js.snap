// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title Component should render title 1`] = `
<div>
  <h1
    className="title-container__title"
  >
     Weather Finder 
  </h1>
  <h3
    className="title-container__subtitle"
  >
     Find out temperature, humidity and ohter weather conditions. 
  </h3>
</div>
`;
